view/frontend/web/js/view/payment/method-renderer/gr4vy-method.js

```diff
@@ -117,7 +118,12 @@ define(
                             button_collection[0].style.display = 'none';
 
                             var placeholder_collection = document.getElementsByClassName('gr4vy-placeholder');
-                            placeholder_collection[0].innerHTML += $t('<span class="gr4vy-checkout-notice">Payment method is not available. Please contact us for support</span>');
+                            if (total_mismatch) {
```

Comment:
See above.  Instead of showing this error, we want to send the request anyway, without cart items